storybook/storybook-react/src/components/Heading/Heading.stories.tsx

```diff
@@ -11,21 +11,51 @@ const meta = {
   component: Heading,
   args: {
     children: 'Jouw typograaf biedt mij zulke exquise schreven!',
-    level: 1,
-    size: 'level-1',
+    level: undefined,
```

Comment:
Why do we need these?

Reply:
I thought it was a way to keep the code view clean of props, but it doesn't actually seem to be necessary. Not sure why I thought it was 😅 I'll remove it from the main and `InvertedColor` stories